### ISSUE TITLE
Save timezones for DateTime types

### ISSUE THREAD

**Issue by mehertz** [man-group/ArcticDB#189]

**Comment by lwfitzgerald** [comment]
I'm not sure this is completely correct - My understanding is that tz-aware datetimes are **converted** to UTC then made naive, rather than them being assumed to be UTC.

Given the amount of existing code that probably relies on the current behaviour, I think some sort of opt in/out switch would be required to enable retaining of the TZ info.

**Comment by chazkii** [comment]
Any rough ETA for this enhancement?

**Comment by tlk3** [comment]
Am I correct in assuming library.write() does not save tz info? When I write this dataframe to arctic, it strips timezone. 

`                         market_open              market_close
2023-08-24 2023-08-24 13:30:00+00:00 2023-08-24 20:00:00+00:00
2023-08-25 2023-08-25 13:30:00+00:00 2023-08-25 20:00:00+00:00
2023-08-28 2023-08-28 13:30:00+00:00 2023-08-28 20:00:00+00:00
2023-08-29 2023-08-29 13:30:00+00:00 2023-08-29 20:00:00+00:00
2023-08-30 2023-08-30 13:30:00+00:00 2023-08-30 20:00:00+00:00
2023-08-31 2023-08-31 13:30:00+00:00 2023-08-31 20:00:00+00:00
2023-09-01 2023-09-01 13:30:00+00:00 2023-09-01 20:00:00+00:00
2023-09-05 2023-09-05 13:30:00+00:00 2023-09-05 20:00:00+00:00
2023-09-06 2023-09-06 13:30:00+00:00 2023-09-06 20:00:00+00:00 
`

`
In [29]: spy.exchange.library.write('NYSE_SCHEDULE:D', df)
Out[29]: VersionedItem(symbol='NYSE_SCHEDULE:D', library='schedule', data=n/a, version=4, metadata=None, host='LMDB(path=/home/code/stratbot2-dev/lmdb)')

In [30]: spy.exchange.library.head('NYSE_SCHEDULE:D').data
Out[30]:
                   market_open        market_close
2023-08-24 2023-08-24 13:30:00 2023-08-24 20:00:00
2023-08-25 2023-08-25 13:30:00 2023-08-25 20:00:00
2023-08-28 2023-08-28 13:30:00 2023-08-28 20:00:00
2023-08-29 2023-08-29 13:30:00 2023-08-29 20:00:00
2023-08-30 2023-08-30 13:30:00 2023-08-30 20:00:00
`

**Comment by qc00** [comment]
> Am I correct in assuming library.write() does not save tz info?

You're correct. Each timestamp is saved as a single 64-bit int only, so the timezone is not preserved.
